### PR TITLE
DM-7692: Create from validate_drp 

### PR DESCRIPTION
**Pull request by timj**

### CODE REVIEW

File: COPYRIGHT [hunk 1]
```diff
@@ -1 +1 @@
-Copyright University of Pittsburgh and AURA/LSST (2015-2016)
+Copyright (C) 2016 Association of Universities for Research in Astronomy, Inc.
```
Comment (timj) [inline COPYRIGHT:1]: Did all the Pittsburgh code get left behind in validate_drp?

File: README.md [hunk 2]
```diff
@@ -2,5 +2,48 @@
...
+Then, to install this `validate_base` package from source:
+
+```
+git clone https://github.com/lsst/validate_base
+cd validate_base
+eups declare -r . -t $USER validate_base git
+setup -r . -t $USER
+scons
+```
+
```
Comment (timj) [inline README.md:31]: They can also install it using `rebuild` or, if you are included in the weeklies, `eups distrib install`.

File: COPYRIGHT (hunk 1 above)
Comment (jonathansick) [reply COPYRIGHT:1]: Yep.

File: README.md (hunk 2 above)
Comment (jonathansick) [reply README.md:31]: Maybe I could re-title the section as "Install from source"? I was trying to answer the user story of: "if I clone this repo, how do I use my clone?"

File: ups/validate_base.table [hunk 3]
```diff
@@ -0,0 +1,8 @@
+setupRequired(pyyaml)
+setupRequired(numpy)
+setupRequired(astropy)
```
Comment (timj) [inline ups/validate_base.table:4]: Should probably depend on `python` as well.

Comment (timj) [reply ups/validate_base.table:4]: Also, you do need `sconsUtils`.

File: ups/validate_base.table [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+setupOptional(base)
```
Comment (timj) [inline ups/validate_base.table]: I'd drop this completely. You don't even really need it optionally. No extra functionality is enabled if it is there.

File: README.md (hunk 2 above)
Comment (timj) [reply README.md:31]: Sometimes people are interested in knowing how to short circuit the install. You see people say "use pip install" in READMEs. You are covering the "I want to make changes, how do I build this?" information.

File: ups/validate_base.table (hunk 4 above)
Comment (jonathansick) [reply ups/validate_base.table]: Ah ok. So when I wrap `import lsstimport` in a try/except, `validate_base` isn't even optionally depending on `base`.

Comment (timj) [reply ups/validate_base.table]: Correct. It only needs to work if other packages have been setup that need it. Those packages will have enabled `base` so the import will work.

**Comment by jonathansick** [comment]
Thanks @timj for the suggestions. Everything should be covered. 👍

**Pull request merged**